### PR TITLE
docs(react): added fallback route to routing guide

### PR DESCRIPTION
**Pull request by eric-horodyski**

### CODE REVIEW

File: src/pages/react/navigation.md [hunk 1]
```diff
@@ -97,6 +97,42 @@ The `DashboardPage` above shows a users list page and a details page. When navig
...
+    <IonRouterOutlet>
+      <Route exact path={match.url} component={UsersListPage} />
+      <Route path={`${match.url}/users/:id`} component={UserDetailPage} />
+      <Route render={() => <Redirect to={match.url} />} />
+    </IonRouterOutlet>
+  );
+};
+```
+
+Here, we see that in the event a location does not match the first two `Route`s it will redirect the Ionic React app to the `match.url` path.
```
Comment (liamdebeasi) [inline src/pages/react/navigation.md:120]:
```suggestion
Here, we see that in the event a location does not match the first two `Route`s, `IonRouterOutlet` will redirect the Ionic React app to the `match.url` path. 
```

File: src/pages/react/navigation.md [hunk 2]
```diff
@@ -97,6 +97,42 @@ The `DashboardPage` above shows a users list page and a details page. When navig
 
 An `IonRouterOutlet` should only contain `Route`s or `Redirect`s. Any other component should be rendered either as a result of a `Route` or outside of the `IonRouterOutlet`.
 
+## Fallback Route
+
+A common routing use case is to provide a "fallback" route to be rendered in the event the location navigated to does not match any of the routes defined. 
+
+We can define a route within an `IonRouterOutlet` to act as our "fallback" route -- provided no `path` is provided and it is the last `Route` defined.
```
Comment (liamdebeasi) [inline src/pages/react/navigation.md:104]: "provided no path is provided" sounds a bit odd since we use "provided" twice. Can we change to make clearer?

**Review by liamdebeasi — CHANGES_REQUESTED** [review]
Two small comments, otherwise looks great!

File: src/pages/react/navigation.md (hunk 1 above)
Comment (liamdebeasi) [reply src/pages/react/navigation.md:120]: I am assuming "it" refers to IonRouterOutlet. Let me know if that is not the case.

**Review by elylucas — APPROVED** [review]
agree with liams comments, but lgtm after that

File: src/pages/react/navigation.md [hunk 3]
```diff
@@ -117,12 +116,12 @@ const DashboardPage: React.FC<RouteComponentProps> = ({match}) => {
 };
 ```
 
-Here, we see that in the event a location does not match the first two `Route`s it will redirect the Ionic React app to the `match.url` path. 
+Here, we see that in the event a location does not match the first two `Route`s. The `IonRouterOutlet` will redirect the Ionic React app to the `match.url` path.
```
Comment (liamdebeasi) [inline src/pages/react/navigation.md:119]: This first part should be reworded because it is not a complete sentence.

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

File: src/pages/react/navigation.md (hunk 3 above)
Comment (eric-horodyski) [reply src/pages/react/navigation.md:119]: D'oh! Let me fix it 😄

**Review by liamdebeasi — APPROVED** [review]
Looks great!

**Pull request merged**